pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Overview
+Provide a brief description of what this PR does, and why it's needed.
+
+### Does this PR relate to an RChain JIRA issue? 
+If applicable, add link to corresponding JIRA issue.
+
+### Complete this checklist before you submit the PR
+- [ ] This PR contains no more than 200 lines of code.
+- [ ] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
+- [ ] You have someone in mind to assign for review. Make this assignment after submitting the PR.
```

Comment:
You must assign someone for review.

Reply:
Thanks, @MParlikar. Since you can't assign someone for review until you take the next step in the PR process, and you need to go through this checklist before you go through the process. Would it be okay to edit so it reads: "You have determined who you will assign to review this PR. You must assign someone for review."

Reply:
You can add a reviewer before clicking the "submit" button.

Reply:
Are you sure Henry? I was not able to do that until I was added as a Collaborator on the project.